examples/110-formula-usage/example-formula-exec.md

```diff
@@ -39,9 +39,7 @@ This formula echos a value to stdout.
 	},
 	"context": {
 		"context.v1": {
-			"warehouses": {
-				"tar:57j2Ee9HEtDxRLE6uHA1xvmNB2LgqL3HeT5pCXr7EcXkjcoYiGHSBkFyKqQuHFyGPN": "https://dl-cdn.alpinelinux.org/alpine/v3.15/releases/x86_64/alpine-minirootfs-3.15.0-x86_64.tar.gz"
-			}
+			"warehouses": {}
```

Comment:
I think a little snippet of prose in this area to comment on this might be good.

Perhaps:

> Note that for the "warehouses" field to be empty is a little unusual -- usually, URLs are present there, which warpforge will use if it needs to fetch any inputs.  In this demo, it can be blank, because our test harness pre-populates caches in the test environment, so the tests work without any need for URLs here.

Reply:
... actually, gosh, now that I think about it a second longer -- maybe we should actually put the URLs back in here, even.

This is the "examples" folder.

It's good for our tests to run without network... but these should still be examples ready for someone else to copy-paste.  That means warehouse address URLs should be textually present.

Our tests in CI should probably work no-net, but... a cache hit "happens" to accomplish that.

Maybe a stretch goal should be: we add some flags to the test harnesses that will also run them _without_ the no-net setup process, so as to do a load-bearing check that these URLs are in fact live.  We wouldn't run those tests when developing, but having them available as a health check would seem reasonable.

Reply:
They have been readded everywhere *except* the CLI runformula example. That is the only example that actually required network. I have added some prose to explain.

Ideally, would be nice to have these behind a net-flag but for now this seems like an acceptable tradeoff.

Reply:
Now behind the `-testutil.offline` flag.